students/jvirtue/lesson03/mailroom.py

```diff
@@ -0,0 +1,86 @@
+#Lesson 3 Assignment 4
+#Mailroom Part Assignment
+#Jason Virtue 01/26/2019
+#UW Self Paced Python Course
+
+
+#Thank You Letter Section
+def Send_Thanks(db_name):
```

Comment:
per python style guide function names should be snake_case -> `def send_thanks`

Reply:
change to snake_case for all functions